eternalcore-core/src/main/java/com/eternalcode/core/feature/burn/messages/PLBurnMessages.java

```diff
@@ -0,0 +1,18 @@
+package com.eternalcode.core.feature.burn.messages;
+
+import com.eternalcode.multification.notice.Notice;
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Accessors(fluent = true)
+public class PLBurnMessages extends OkaeriConfig implements BurnMessages {
+
+    @Comment(" ")
```

Comment:
Add a comment related to the placeholders that accept messages, or remove this empty comment entirely. The section currently looks like this:

```yaml
burn:
  
  burnedSelf: "..."

  burnedOther: "  "
```

but it should look like this:

```yaml
burn:
  burnedSelf: "..."

  burnedOther: "  "
```


Same for ENBurnMessages.